app/frontend/tests/unit/services/acronymService.spec.js

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+
+import acronymService from '@/services/acronymService';
+import { ApiRoutes } from '@/utils/constants';
+
+const mockInstance = axios.create();
+const mockAxios = new MockAdapter(mockInstance);
+const acronym = 'XXX';
+
+jest.mock('@/services/interceptors', () => {
+  return {
+    getokAxios: () => mockInstance
+  };
+});
+
+describe('getAcronym', () => {
+  const data = {
+    acronymId: '123-456',
+    acronym: 'MSSC',
+    name: '',
+    description: 'None Entered',
+    permissionWebade: false,
+    permissionWebadeNrosDms: false,
+    createdAt: '2019-08-15T20:14:41.966Z',
+    updatedAt: '2019-08-15T20:14:41.966Z',
+    deletedAt: null
+  };
```

Comment:
As a note, we probably don't need this to be "complete" data instance as we don't actually test for pieces of data in there - only that this specific object is the body return. No action required.